examples/enclave_manager/tcf_enclave_manager/enclave_manager.py

```diff
@@ -229,7 +274,7 @@ def create_enclave_signup_data():
 
     return enclave_signup_data
 
-# -----------------------------------------------------------------
+    # -----------------------------------------------------------------
```

Comment:
This should not be indented. Was this a mistake? Or on purpose?

Reply:
Corrected